packages/amplify-category-notifications/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@aws-amplify/amplify-category-notifications",
-  "version": "2.26.21",
+  "version": "2.26.22-s3-bucket-ssl.0",
```

Comment:
these versioning changes should be reverted. Main shouldn't have tagged release versions